examples/1-hello-world/README.md

```diff
@@ -0,0 +1,134 @@
+# Hello World
+
+This is very simple `bin` crate generated by `cargo new`, with no additional
+modification. It is built using the latest stable Rust, as provided by
+[nixpkgs-mozilla](https://github.com/mozilla/nixpkgs-mozilla).
+
+## How it was created
+
+1. Install the `cargo2nix` tool according to the [official instructions], if you
+   haven't already.
+2. Create a new `bin` project with `cargo new hello-world`. If the Rust
+   toolchain is not installed on your system, you could use `nix-shell` like so:
+   ```bash
+   nix-shell -p cargo --run 'cargo new hello-world'
+   ```
+3. Run `cargo generate-lockfile` to create a `Cargo.lock`, if one is not
+   already present.
+4. Run `cargo2nix -f` at the project root to generate a `Cargo.nix`.
+5. Write the `default.nix`, and optionally a `shell.nix`.
+
+[official instructions]: ../../README.md#install
+
+## Building
+
+To compile the `hello-world` binary with Nix, run:
+
+```bash
+nix-build -A package
```

Comment:
recommend `my-package` to cue users that this identifier is bound, not a keyword

Reply:
Would you mind elaborating on what you mean by this, exactly? `package` isn't a language keyword, and I imagine it's pretty self-explanatory what `-A package` does since we assume basic high-level knowledge of Nix for incoming users.